### ISSUE TITLE
Create README for caesar-cipher interactive

### ISSUE THREAD

**Issue by JackMorganNZ** [uccser/cs-field-guide#288]
A `README.md` file is required for this interactive. An example README is below:

```
# Packet Attack Interactive

**Original Author:** Sam Jarman  
**Modified By:** Jack Morgan

This interactive is created for illustrating network issues to the user, but delaying/corrupting/killing packets of data.

This interactive currently does not work in Google Chrome when viewed locally (by opening `index.html`). Either view this interactive in a different browser when viewing offline, or view it online in Google Chrome.

## Required Files

- [jQuery](https://jquery.com/) (loaded from base-files folder)
- [Phaser](https://github.com/photonstorm/phaser)
- [Moment](http://momentjs.com/)
- [Moment duration format](https://github.com/jsmreese/moment-duration-format)
- [Lodash Compat](https://github.com/lodash/lodash-compat)

## Future plans

- There are still some references of birds and
  flying from the original messenger birds
  prototype. These should be updated.
- Allow full screen in phaser, but HTML button
  below interactive.
- Adapt code to align with standard Phaser
  function breakdown (preload/create/etc).
```

**Comment by Jamesernator** [comment]
Closed with 0ba6c472bd93c97cdcaebe7d9b731f6115e6fce1

**Comment by JackMorganNZ** [comment]
There is an issue with the PR for this issue, I've made a comment in the code for you.

If you don't have time to fix it before release, I can fix it (but have many other things to do before then).

**Comment by Heidirose** [comment]
@JackMorganNZ what time is the release? I think @Jamesernator has classes all afternoon.

**Comment by JackMorganNZ** [comment]
I've been given a lot of issues to work through and have teaching soon so probably tomorrow. If he could do it that would be great, though I can do it tomorrow easily (just one line deletion).

**Comment by Jamesernator** [comment]
Fixed with 9c3b3013be3c15bfa520a3707d426724b39dd3fa
